docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Attribution"
+sidebar_position: 400
+hide_title: true
+---
+
+```mdx-code-block
+import ThemedImage from '@theme/ThemedImage';
+import Badges from '@site/src/components/Badges';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+<Badges badgeType="dbt-package Release" pkg="attribution">&nbsp;</Badges>&nbsp;
+<Badges badgeType="Maintained"></Badges>
+
+
+# Snowplow Attribution Package
+​
+**The package source code can be found in the [snowplow/dbt-snowplow-attribution repo](https://github.com/snowplow/dbt-snowplow-attribution), and the docs for the [macro design are here](https://snowplow.github.io/dbt-snowplow-attribution/#/overview/snowplow_attribution).
+​
+:::note
+
+Although there is no direct dependency, by default the attribution package is dependent on the `snowplow_unified_views` as well as the `snowplow_unified_conversions` table created by the [snowplow_unified](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model/index.md) dbt package. As long as the field names are matched you can use other source tables, too.
+
+:::
+
+## Overview
+​
+The Snowplow Attribution dbt package produces a set of incremental derived tables to provide a basis for in-depth **Marketing Attribution Analysis** on an ongoing basis. It allows you to attribute the value of a conversion to one or more channels and campaigns depending on the conversion pathway. As a result, it becomes possible to determine the revenue per pathway (channel or campaign), as well as ROAS (Return On Ad Spend - the amount of revenue that is earned for every dollar spent on advertising) once you have cost data for each marketing channel or campaign.
+
+This package consists of a series of dbt models that produce the following tables:
+
+- `snowplow_attribution_paths_to_conversion`: Customer id and the the paths the customer has followed that have lead to conversion
+- `snowplow_attribution_campaign_attributions`: By campaign path and conversion level incremental table that attributes the conversion value based on various algorythms
+- `snowplow_attribution_channel_attributions`: By channel path and conversion level incremental table that attributes the conversion value based on various algorythms
+- `snowplow_attribution_overview`: The user defined report view (potenitally showing ROIs)
```

Comment:
Perhaps we use ROAS here instead of ROI as they have slightly different definitions, ROI looks at profit, rather than revenue. Also typo in 'potenitally'